frontend-react/e2e/mocks/deliveries.ts

```diff
@@ -804,3 +804,885 @@ export const MOCK_GET_DELIVERIES_IGNORE_FULL_ELR = [
         originalIngestion: null,
     },
 ];
+
+export const MOCK_V1_GET_DELIVERIES_AK = {
+    data: [
+        {
```

Comment:
This object does not match the endpoint response.
Should look something like this:
```
"data": [
        {
            "deliveryId": "2591",
            "createdAt": "2024-07-01T19:46:01.297Z",
            "expiresAt": "2024-07-31T19:46:01.297Z",
            "receivingOrgSvcStatus": "inactive",
            "reportId": "08dc3165-9134-4fb8-8664-340e7f428a5c",
            "topic": "covid-19",
            "reportItemCount": 1,
            "fileType": "HL7",
            "receiver": "ignore.HL7",
            "fileName": "ignore.HL7%2F08dc3165-9134-4fb8-8664-340e7f428a5c.hl7"
        },
```

Reply:
Thank you -- updated!